docs/appservice/_category_.json

```diff
@@ -1,6 +1,6 @@
 {
   "label": "Appservice",
-  "position": 1,
```

Comment:
This was Tbh because Alphabetical order looks nice. But it makes sense to change it.

Reply:
Yeah i don't want newbies reading the appservice first